scripts/run_demo.sh

```diff
@@ -47,6 +47,9 @@ if [ $wait -gt 60 ]; then
   exit -1
 fi
 
+docker run --env MONGO_DB="mongodb://mongo:27017/graphql" --name=populateDB --network=anubis_envoymesh orchestracities/configuration-api node main/mongo/populateDB.js
```

Comment:
This leads to a `docker: Error response from daemon: network anubis_envoymesh not found`